tests/sentry/api/test_organization_missing_members.py

```diff
@@ -86,9 +99,46 @@ def test_filters_old_commits(self):
         ]
 
     def test_no_authors(self):
-        org = self.create_organization()
+        org = self.create_organization(owner=self.create_user())
         self.create_member(user=self.user, organization=org, role="manager")
 
         response = self.get_success_response(org.slug)
         assert response.data[0]["integration"] == "github"
         assert response.data[0]["users"] == []
+
+    def test_not_common_domain(self):
+        user = self.create_user(email="owner@exampletwo.com")
+        self.create_member(
+            organization=self.organization,
+            user=user,
+            role="owner",
+        )
+        OrganizationMember.objects.filter(
+            organization_id=self.organization.id, role="owner", user_id=user.id
+        ).update(user_email="owner@exampletwo.com")
```

Comment:
Isn't this email already `owner@exampletwo.com`? What's this doing

Reply:
`email` is not the same as `user_email`. for some reason in test cases this has to be explicitly set. we check for `user_email` in the endpoint because that's the email associated with the user, while email is only set whilst the user has yet to accept the invite